spec/controllers/miq_report_controller/trees_spec.rb

```diff
@@ -107,11 +109,13 @@
         expect(response.body).not_to include(other_group.name)
       end
 
+      let(:miq_widget) { FactoryBot.create(:miq_widget) }
+
       it 'renders show of Dashboards in Dashboards tree' do
         ApplicationController.handle_exceptions = true
 
         MiqWidgetSet.seed
-        widget_set = FactoryBot.create(:miq_widget_set, :group_id => user.current_group.id)
+        widget_set = FactoryBot.create(:miq_widget_set, :set_data => {:col1 => [miq_widget.id]}, :owner_id => user.current_group.id, :group_id => user.current_group.id)
```

Comment:
there are a bunch of places where the `owner_type` is not set - for this reason the group was passed in. if you pass owner (or owner_id and owner_type) then group will get set correctly